c-dependencies/js-compute-runtime/xqd-world/xqd_world.h

```diff
@@ -0,0 +1,392 @@
+#ifndef __BINDINGS_XQD_WORLD_H
+#define __BINDINGS_XQD_WORLD_H
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+
+typedef struct {
+  char *ptr;
+  size_t len;
+} xqd_world_string_t;
+
+typedef uint8_t fastly_error_t;
+
+#define FASTLY_ERROR_UNKNOWN_ERROR 0
+#define FASTLY_ERROR_GENERIC_ERROR 1
+#define FASTLY_ERROR_INVALID_ARGUMENT 2
+#define FASTLY_ERROR_BAD_HANDLE 3
+#define FASTLY_ERROR_BUFFER_LEN 4
+#define FASTLY_ERROR_UNSUPPORTED 5
+#define FASTLY_ERROR_BAD_ALIGN 6
+#define FASTLY_ERROR_HTTP_INVALID 7
+#define FASTLY_ERROR_HTTP_USER 8
+#define FASTLY_ERROR_HTTP_INCOMPLETE 9
+#define FASTLY_ERROR_OPTIONAL_NONE 10
+#define FASTLY_ERROR_HTTP_HEAD_TOO_LARGE 11
+#define FASTLY_ERROR_HTTP_INVALID_STATUS 12
```

Comment:
This is missing the newer error case for exceeding limits (https://github.com/fastly/Viceroy/commit/8ee547b53cadb8366082bd7ca931be0a3a770411):
```suggestion
#define FASTLY_ERROR_HTTP_INVALID_STATUS 12
#define FASTLY_ERROR_LIMIT_EXCEEDED 13
```